test/test-cases/braid-design-system/app/src/App.js

```diff
@@ -7,30 +7,35 @@ import {
   Checkbox,
   Card,
   ChevronIcon,
+  Box,
 } from 'braid-design-system';
+import * as style from './App.treat';
 
 const Theme = loadable.lib(props =>
   import(`braid-design-system/themes/${props.themeName}`),
 );
 
 const noop = () => {};
 
-export default ({ theme: themeName }) => (
-  <Theme themeName={themeName}>
-    {({ default: theme }) => (
-      <BraidProvider theme={theme}>
-        <Text>
-          Hello {themeName} <ChevronIcon inline />
-        </Text>
-        <Card>
-          <Checkbox
-            checked={false}
-            onChange={noop}
-            id="id_1"
-            label="This is a checkbox"
-          />
-        </Card>
-      </BraidProvider>
-    )}
-  </Theme>
-);
+export default ({ theme: themeName }) => {
+  return (
+    <Theme themeName={themeName}>
+      {({ default: theme }) => (
+        <BraidProvider theme={theme}>
+          <Text>
+            Hello {themeName} <ChevronIcon inline />
+          </Text>
+          <Card>
+            <Checkbox
+              checked={false}
+              onChange={noop}
+              id="id_1"
+              label="This is a checkbox"
+            />
+          </Card>
+          <Box className={style.customBox}>Custom content</Box>
```

Comment:
What the purpose of this change? To test treat locally?

Reply:
Yeah just wanted an end-to-end test case. I added this file to test the hot reloading update and figured it wouldnt hurt to keep it.